src/twisted/application/service.py

```diff
@@ -92,13 +92,17 @@ class IService(Interface):
     A service.
 
     Run start-up and shut-down code at the appropriate times.
-
-    @type name:            C{string}
-    @ivar name:            The name of the service (or None)
-    @type running:         C{boolean}
-    @ivar running:         Whether the service is running.
     """
 
+    name = Attribute(
+        "A C{string} which is the name of the service or c{None}.")
```

Comment:
This should be C{None}. with capital C.

But I don't think that this documentation will work.

We should still document them using epydoc as docstrings.

Reply:
There's no such type as "string".  It's either `bytes` or `unicode` and this is the place  to say which.